product_unique_serial/tests/test_for_unicity.py

```diff
@@ -27,93 +27,21 @@
 from copy import deepcopy
 
 from openerp import exceptions
-from openerp.tests.common import TransactionCase
+from openerp.addons.product_unique_serial.tests.test_common import TestCommon
```

Comment:
At my first attempt I tried to do so and I failed to succeed,
now I know why.

Best regards